libkbfs/folder_block_manager.go

```diff
@@ -982,6 +982,10 @@ func (fbm *folderBlockManager) isQRNecessary(
 		fbm.log.CWarningf(ctx, "Couldn't get the current session: %+v", err)
 		return false
 	}
+	// It's ok to treat both MDs written by this process on this
+	// device, and MDs written by other processes (e.g., kbgit) in the
+	// same way.  Other processes are likely to be short-lived, and
+	// probably won't do their own QR, so a conflict is unlikely here.
```

Comment:
That's right.  It will cause conflict resolution to kick in, when normally QR would try to avoid that within the same device.  But because we will have multiple processes acting on behalf of the same device, it's much harder to avoid that now.  But yes, correctness won't be affected.